lib/rubocop/cop/style/hash_syntax.rb

```diff
@@ -49,21 +51,30 @@ def hash_rockets_check(node)
         def ruby19_no_mixed_keys_check(node)
           pairs = *node
 
-          if @force_hash_rockets
```

Comment:
Based on [line 28](https://github.com/daviddavis/rubocop/blob/af0169ac1d5eb02d114727e8ea6232e09062c74b/lib/rubocop/cop/style/hash_syntax.rb#L28), I don't think this line will ever be true here.